agent/cmd/cuda-checkpoint-helper/daemon_protocol_test.cpp

```
@@ -0,0 +1,560 @@
+      std::chrono::seconds(
+          static_cast<uint64_t>(std::numeric_limits<unsigned int>::max()) /
+              1000ULL +
+          1ULL),
+      &timeout_ms, &error));
+}
+
+void TestBoundedOutputCaptureDrainsAndTruncates() {
+  BoundedOutputCapture capture(8);
+  std::string error;
+  assert(capture.Start(&error));
+  const std::string payload(1024 * 1024, 'x');
```

Clang (14.0.6)

[warning] performing an implicit widening conversion to type 'std::basic_string<char>::size_type' (aka 'unsigned long') of a multiplication performed in type 'int'

(bugprone-implicit-widening-of-multiplication-result)